packages/python/plotly/plotly/figure_factory/_dendrogram.py

```diff
@@ -32,7 +32,29 @@ def create_dendrogram(
     :param (ndarray) X: Matrix of observations as array of arrays
     :param (str) orientation: 'top', 'right', 'bottom', or 'left'
     :param (list) labels: List of axis category labels(observation labels)
-    :param (list) colorscale: Optional colorscale for dendrogram tree
+    :param (list) colorscale: Optional colorscale for dendrogram tree. To
```

Comment:
this is probably too detailed an explanation... most people will likely not want to follow the indirection and go read up on what scipy does here. I think just saying "should be 8 colors" is sufficient :)

Reply:
And we should mention that no matter the version of `scipy`, the 7th color in the scale is ignored (note: this is because the `w` key isn't outputted by scipy), and in `scipy` 15+, colors 1,2 and 3 are used twice as often as the others. 🤦